book/website/ethical-research/cultural-change.md

```diff
@@ -0,0 +1,81 @@
+(er-cultural-change)=
+
+# Cultural Change
+
+(er-cultural-change-what)=
+## What is cultural change?
+
+Culture is defined in many ways, one of which is about the social environment in which you partake, or the customs and behaviours that define a group of people.
+Culture has an influence on your views, values, concerns and your identity. 
+Culture can be difficult to change as it is hard to address directly and shared values can be deeply ingrained ({cite:ps}`Kotter2012change`). 
+Instead, norms of behaviour may be easier to change. 
+
+Here, cultural change is defined as inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation.
+
+This section will continue to describe where you will encounter cultural change. 
+This is followed by sub-chapters short summaries of works on cultural change, written by adrienne marie brown, William and Susan Bridges and John P. Kotter, Brian Nosek and a summary of the discussions on alternative models held by _The Turing Way_ community at the June 2024 Book Dash. 
+
+
+```{figure} ../../figures/culture-shift.*
```

Comment:
Hi @Arielle-Bennett I think this should be `../figures` to fix the build issue.
(Oddly though the error in the build log has `../figures` despite it being `../../figures` here puzzling...)

Reply:
@RichardJActon Maybe that is because `jupyter-book` is printing the path relative to the `website` directory?

Reply:
Ah yep that makes sense - error message reads to me more like: 'went looking for the file you said was here: x verbatim' rather than 'went looking for a file you said should have this: <evaluated path>'